src/app/storage.js

```diff
@@ -0,0 +1,97 @@
+var utils = require('./utils');
+
+function Storage (updateFiles) {
+
+  var EDITOR_SIZE_CACHE_KEY = 'editor-size-cache';
+
+  this.rename = function (originalName, newName) {
+    var content = this.get(originalName);
+    this.set(newName, content);
+    this.remove(originalName);
+  };
+
+  this.remove = function (name) {
+    window.localStorage.removeItem(name);
+  };
+
+  this.setEditorSize = function (size) {
```

Comment:
This encapsulates the storage/retrieval of the editor size from localStorage. In the future, we could create wrapper methods in editor.js if that makes more sense.